include/avt_vimba_camera/mono_camera.h

```diff
@@ -43,19 +43,22 @@
 #include <camera_info_manager/camera_info_manager.h>
 #include <image_transport/image_transport.h>
 #include <dynamic_reconfigure/server.h>
-
+#include <cav_msgs/SystemAlert.h>
+#include <cav_msgs/DriverStatus.h>
 #include <string>
 
 namespace avt_vimba_camera {
 class MonoCamera {
  public:
   MonoCamera(ros::NodeHandle& nh, ros::NodeHandle& nhp);
   ~MonoCamera(void);
-
+ void updateCameraStatus();
+ short int cam_status;
```

Comment:
A better date type should be ```unsigned char```, because it is defined as ```UNIT8```.
Usually we need to make it private and use getters and setters as I mentioned in last peer review comments.